### DISCUSSION TITLE
I delete the dag and it is recreated

### DISCUSSION THREAD

**Discussion by leticiajsx** [apache/airflow#39761] [General]
### Apache Airflow version

2.9.1

### What happened?

I click on the button to delete the dag, the dag disappears from the list and the message appears informing that it will soon be deleted, after a minute the dag returns to the list of dags

![image](https://github.com/apache/airflow/assets/86029029/e541906c-3b8b-4b79-b7a4-2f1e4a844562)

![image](https://github.com/apache/airflow/assets/86029029/232971f9-db36-430f-935f-6fd591df0442)

![image](https://github.com/apache/airflow/assets/86029029/47f7f313-9404-46bd-9a76-7de7c6b75bee)

### What you think should happen instead?

I click on the button to delete the dag, the dag disappears from the list and the message appears informing that it will soon be deleted, the dag no longer returns to the list of dags

### How to reproduce

Click on the button to delete the dag, you will notice the dag disappear from the list and the message will appear informing you that it will soon be deleted, after a minute the dag will appear again in the list of dags

### Operating System

windows

### Deployment

Other

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by KamilKandzia** [comment]
Did you deleted specific DAG from the dags folder?

**Comment by Taragolis** [comment]
You have to delete DAG from the your DAG directory, otherwise it will recreated during next [DAG File Processing](https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/dagfile-processing.html).

Webserver do not have access to the DAG directory, even if it has there is not possible to resolve which file should be deleted

**Comment by chaityacshah** [comment]
@leticiajsx are you good with this?
